Library/Homebrew/utils/git.rb

```diff
@@ -14,6 +20,31 @@ def last_revision_commit_of_file(repo, file, before_commit: nil)
     out.chomp
   end
 
+  def last_revision_commit_of_files(repo, files, before_commit: nil)
+    args = [
+      if before_commit.nil?
+        "--skip=1"
+      else
+        before_commit.split("..").first
+      end,
+    ]
+
+    # git log output format:
+    #   <commit_hash>
+    #   <file_path1>
+    #   <file_path2>
+    #   ...
+    # return [<commit_hash>, [file_path1, file_path2, ...]]
+    cmd = [
+      HOMEBREW_SHIMS_PATH/"scm/git", "-C", repo, "log",
+      "--pretty=format:%h", "--abbrev=7", "--max-count=1",
+      "--diff-filter=d", "--name-only", *args, "--", *files
+    ]
+    out, = Open3.capture3(cmd.join(" "))
```

Comment:
Why join these?

Reply:
I tried not joining as well as putting them as arguments directly but the command didn't work. Will investigate further.

Reply:
Works now. I must have forgotten to unpack earlier.